src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -22,13 +22,15 @@ public class LinuxAppServiceFileLogger
         private readonly IFileSystem _fileSystem;
         private readonly CancellationTokenSource _cancellationTokenSource;
         private Task _outputTask;
+        private int _currentFlushFrequencySeconds;
 
-        public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IFileSystem fileSystem, bool startOnCreate = true)
+        public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IFileSystem fileSystem, bool startOnCreate = true, bool logBackoffEnabled = false)
         {
             _logFileName = logFileName;
             _logFileDirectory = logFileDirectory;
             _logFilePath = Path.Combine(_logFileDirectory, _logFileName + ".log");
             _buffer = new BlockingCollection<string>(new ConcurrentQueue<string>());
+            _currentFlushFrequencySeconds = logBackoffEnabled ? 1 : 30;
```

Comment:
Rather than duplicate 30 here, I believe the intent is to use MaxFlushFrequencySeconds? I.e. if MaxFlushFrequencySeconds was set to a differnet value, you'd want to use that here as well.

Reply:
I replaced 30 with MaxFlushFrequencySeconds. Thank you for the callout. I am not sure that I understand how moving to ProcessLogQueue would simplify the code here.